KIMI.md

```
@@ -0,0 +1,306 @@
+
+
+
+## Update Check
+
+
+
+On the first message of each session, run the update checker silently:
+
+
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)